src/host/CursorBlinker.cpp

```diff
@@ -82,10 +82,8 @@ void CursorBlinker::TimerRoutine(SCREEN_INFORMATION& ScreenInfo) const noexcept
     auto& cursor = buffer.GetCursor();
     auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
     auto* const pAccessibilityNotifier = ServiceLocator::LocateAccessibilityNotifier();
-    const auto inConpty{ gci.IsInVtIoMode() };
 
-    // GH#2988: ConPTY can now be focused, but it doesn't need to do any of this work either.

```

Comment:
IS this short circuit no longer true?

Reply:
Instead of doing a short-circuit there, I'm earl-returning in the enable-blinking function further down below. This prevents the creation of the timer in the first place.